tikv-configuration-file.md

```diff
@@ -1117,8 +1117,8 @@ Configuration items related to `rocksdb.defaultcf`, `rocksdb.writecf`, and `rock
 ### `compaction-style`
 
 + Compaction method
-+ Optional values: `"level"`, `"universal"`
-+ Default value: `"level"`
++ Optional values: `0` (`Level`), `1` (`Universal`), `2` (`Fifo`)
```

Comment:
@tabokie @TomShawn how about adding this description?

```suggestion
+ Optional values (only numbers can be accepted): `0` (`Level`), `1` (`Universal`), `2` (`Fifo`)
```

Reply:
No objection.

Reply:
LGTM